### DISCUSSION TITLE
유니티로 만든 제페토 월드에도 아이템 등록이 가능한가요?

### DISCUSSION THREAD

**Discussion by yoon-hw22** [naverz/zepeto-studio-kor#965] [5. Q&A]
유니티 가이드에서는 월드 스튜디오에서 상품을 등록하고 제페토 승인 후 유니티에서 연동을 하면 아이템을 사용할 수 있다고 나와있는데, 혹시 이 기능은 제페토 build it 에서만 제공되는 기능인가요? 

유니티로 개발한 제페토 월드에는 item box ui가 없는데 그럼에도 이렇게 월드에 상품을 등록하면 아이템 구매와 저장, 사용이 가능한지 궁금합니다. 

https://studio.zepeto.me/kr/guides/making-world-products#

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 말씀주신 월드 상품은 유니티 월드에서 사용 가능한 인앱 아이템입니다.
IWP라는 용어를 사용하고 있으며 멀티플레이 월드에서 직접 구현을 통해 판매하실 수 있습니다. 
아래 가이드 및 영상을 참고하여 주세요.

https://studio.zepeto.me/guides/in-world-purchases
https://youtu.be/d-MaqjP-vjA

감사합니다.

**Reply by yoon-hw22** [reply]
답변 감사합니다. 영상에서는 아이템 구매 시 캐릭터의 스피드가 바뀌는 것으로 설정이 되어있는데, 이런거 말고 오브젝트가 나타나거나, 개인 인벤토리로 연결되거나 하는 것은 유니티 커스텀 환경에서는 불가능한 것이죠?

**Reply by Zepeto-TechEvangelist** [reply]
상품 구매 성공도 일종의 월드 내 이벤트 호출이라고 생각하시면 개념 잡기 쉬울것 같습니다. 
따라서 상품 구매 이후 일어나는 이벤트들은 유니티에서 구현 가능한 범위 내에서는 무엇이든 가능합니다.
예를 들어 성능을 업그레이드하거나, 오브젝트 외관을 바꿔주거나, 특정한 오브젝트를 생성하거나, 인 월드 재화를 늘려주는 기능이 가능합니다.
구현 사례로는 공식 월드 중 슬라임 파티의 사례를 보시면 도움이 될 것입니다.
